game/script-greetings.rpy

```diff
@@ -874,6 +874,49 @@ label greeting_love_plus_dull_moment:
 
     return
 
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._greeting_database,
+            label="greeting_love_plus_cant_get_enough",
+            unlocked=True,
+            affinity_range=(jn_affinity.LOVE, None)
+        ),
+        topic_group=TOPIC_TYPE_GREETING
+    )
+
+label greeting_love_plus_cant_get_enough:
+    n 4ccsbg "Well,{w=0.2} well,{w=0.2} well.{w=0.75}{nw}"
+    extend 3fsqbg " And just look who decided to show up today.{w=0.75}{nw}"
+    extend 3fsqsm " Ehehe."
+    n 1fcsbgl "Well,{w=0.2} what can I say?{w=0.75}{nw}"
+    extend 2fchgnl " Guess you really {i}can't{/i} get enough of me,{w=0.2} huh?"
```

Comment:
I noticed this greeting is really similar to an already existing one. To be precises the `greeting_affectionate_enamored_just_cant_stay_away` greeting. I don't think it is bad to have reappearing speak patterns, but I don't see something that justify love affinity, when we already have something similar for lower affinity levels than that.

Reply:
Sure, I can adjust this to be a little more deserving of the affinity.

Reply:
The only thing I'd note with the existing greeting: the affinity ranges are pretty tight on these, so a player on LOVE+ affinity would only get this new one, and would no longer see the `greeting_affectionate_enamored_just_cant_stay_away` greeting (as this stops being applicable at ENAMORED).

Reply:
This has been resolved, reworked the greeting into something a bit more distinct: `greeting_love_plus_nat_dragged_in`